website/docs/usage/models.md

```diff
@@ -365,15 +365,30 @@ pipeline package can be found.
 To download a trained pipeline directly using
 [pip](https://pypi.python.org/pypi/pip), point `pip install` to the URL or local
 path of the wheel file or archive. Installing the wheel is usually more
-efficient. To find the direct link to a package, head over to the
-[releases](https://github.com/explosion/spacy-models/releases), right click on
-the archive link and copy it to your clipboard.
+efficient.
+
+> #### How to Find Download URLs {#download-urls}
+>
+> Pretrained pipeline distributions are hosted on
+> [Github Releases](https://github.com/explosion/spacy-models/releases), and you
+> can find download links there, as well as on the model page. You can also get download URLs directly
+> from the command line by using `spacy info` with the `--url` flag, which may be useful for automation.
+>
+> ```bash
+> spacy info en_core_web_sm --url
+> ```
+>
+> This command will print the download URL for the latest version of a pipeline
+> compatible with the version of spaCy you're using. Note that in order to look up the compatability information an Internet connection is required.
```

Comment:
Do we capitalize internet?

Reply:
I usually do, but it looks like the two other mentions in the docs don't, so I can change it.

Reply:
Changed.